<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ZS-DSK-0014" ignored="0" name="Poc_test" tests="3" failures="1" timestamp="10 Mar 2020 05:42:18 GMT" time="72.344" errors="0">
  <testcase name="testAddDocument" time="16.119" classname="TestPatrixmaven.TestAddDocument"/>
  <testcase name="testCaseCreation" time="41.117" classname="TestPatrixmaven.TestcaseCreation">
    <failure type="java.lang.AssertionError" message="Case not created">
      <![CDATA[java.lang.AssertionError: Case not created
	at TestPatrixmaven.CaseCreation.caseCreation(CaseCreation.java:99)
	at TestPatrixmaven.TestcaseCreation.testCaseCreation(TestcaseCreation.java:34)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testCaseCreation -->
  <testcase name="testDocumentPreview" time="14.737" classname="TestPatrixmaven.TestDocumentPreview"/>
</testsuite> <!-- Poc_test -->
